content/blog/rlang-1-0-0-errors/index.Rmd

```diff
@@ -0,0 +1,179 @@
+---
+output: hugodown::hugo_document
+
+slug: rlang-1-0-0-errors
+title: New error style coming up in rlang 1.0.0
+date: 2021-12-22
+author: Lionel Henry
+description: >
+    rlang 1.0.0 is near and introduces a new style of error display. Feedback welcome!
+
+photo:
+  url: https://unsplash.com/photos/-eDpBjt6UL0
+  author: Bryan Goff
+
+categories: [package]
+tags: []
+---
+
+<!--
+TODO:
+* [x] Look over / edit the post's title in the yaml
+* [x] Edit (or delete) the description; note this appears in the Twitter card
+* [x] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [x] Find photo & update yaml metadata
+* [x] Create `thumbnail-sq.jpg`; height and width should be equal
+* [x] Create `thumbnail-wd.jpg`; width should be >5x height
+* [x] `hugodown::use_tidy_thumbnails()`
+* [x] Add intro sentence, e.g. the standard tagline for the package
+* [/] `usethis::use_tidy_thanks()`
+-->
+
+[rlang](https://rlang.r-lib.org/) 1.0.0 is getting ready for release and we'd like to get your feedback on the new style of error messages featured in this release.
+
+The rlang package provides several low-level frameworks for the tidyverse. The 1.0 release focuses on one of these frameworks, __rlang errors__. This set of tools to signal and display errors gets a substantial overhaul. The three main changes to rlang errors that we'll review in this blog post are:
+
+1. Fully committing to the display of errors as bullet lists
+1. Including the erroring function call by default, as in base R
+1. Embracing chained errors to represent contextual information
+
+If you'd like to try the new error style on your computer, install the development versions of rlang and dplyr (the latter needs to be adapted to the new error style) from github with:
+
+```{r, eval = FALSE}
+pak::pkg_install(c("r-lib/rlang", "tidyverse/dplyr"))
+```
+```{r setup, message = FALSE}
+library(rlang)
+library(dplyr)
+```
```

Comment:
I think it'd be useful to start with a before and after (maybe from dplyr?) that illustrates the main differences. Should also note the change in colour (i.e. error messages aren't all red any more)